.github/workflows/create-release-artifacts.yml

```diff
@@ -79,11 +79,13 @@ jobs:
         shell: bash
         run: |
           ./.github/scripts/install_deps.sh macos
+      - name: Build libpng
+        run: |
+          ./.github/scripts/build_libpng.sh
       # We force linking libpng statically; the other libs are provided by macOS itself
```

Comment:
This comment looks outdated, since the `--static`s are gone.

Reply:
It's still relevant about `-lz`—explaining why we're not linking statically to it.

Reply:
Do you mean that `PNGLDLIBS="libpng-staging/lib/libpng.a -lz"` forces libpng to link statically?

Reply:
Yes, since it's a path to a static archive, and not a dynamic library.